packages/flutter/test/rendering/box_test.dart

```diff
@@ -76,7 +76,7 @@ void main() {
         '   parentData: null\n'
         '   constraints: null\n'
         '   size: MISSING\n'
-        '   decoration:\n'
+        '   decoration: BoxDecoration:\n'
```

Comment:
Previously the name of the decoration class used was omitted. I can make BoxDecoration class use '' as its description 
but that seems a bit more like a bug than a feature.
When you call toString on a box decoration I would think you definitely want 
BoxDecoration(<no decorations specified>)
instead of
<no decorations specified> but perhaps the idea is to try to keep toString for decorations very terse in which case I can switch back to the original behavior.